docs/02-app/02-api-reference/01-components/link.mdx

```diff
@@ -166,9 +181,13 @@ export default function Page() {
 
 ### `prefetch`
 
-**Defaults to `true`.** When `true`, `next/link` will prefetch the page (denoted by the `href`) in the background. This is useful for improving the performance of client-side navigations. Any `<Link />` in the viewport (initially or through scroll) will be preloaded.
+<AppOnly>
 
-Prefetch can be disabled by passing `prefetch={false}`. Prefetching is only enabled in production.
+Prefetching happens when a `<Link />` component enters the user's viewport (initially or through scroll). Next.js prefetches and loads the linked route (denoted by the `href`) in the background to improve the performance of client-side navigations. Prefetching is only enabled in production.
+
+- `null` (default): Prefetch behavior depends on whether the route is static or dynamic. For static routes, the full route will be prefetched. For dynamic routes, the partial route down to the nearest segment with a [`loading.js`](/docs/app/building-your-application/routing/loading-ui-and-streaming#instant-loading-states) boundary will be prefetched.
+- `true`: The full route will be prefetched for both static and dynamic routes.
```

Comment:
Maybe to clarify, the full route means all of the data on the page as well I believe.

Reply:
Good point!